packages/runtime/src/StoryboardFunctionRegistry.ts

```
@@ -149,67 +171,77 @@ export function StoryboardFunctionRegistryFactory({
+                  }
+                : null),
+            }
+          : globalVariables,
+        ArrayConstructor: overrides?.ArrayConstructor,
+        hooks: collector && {
+          beforeEvaluate: collector.beforeEvaluate,
+          beforeCall: collector.beforeCall,
+          beforeBranch: collector.beforeBranch,
+        },
+        debug: !!debuggerOverrides,
+      }) as Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)